curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/5ddb965c65d27e1512d44d9a.md

```diff
@@ -52,8 +52,22 @@ assert.equal(document.querySelector('form')?.id, 'calorie-counter');
 ```
 
 ```css
+:root {
+  --light-text: #f5f6f7;
+  --dark-text: #0a0a23;
```

Comment:
I think a better name for this would be dark-blue. It is also more consistent with your other naming choices. 

```suggestion
  --dark-blue: #0a0a23;
```